RELEASENOTES.md

```diff
@@ -128,13 +129,13 @@ All these actions now uses the selected branch in the **Run workflow** dialog as
 
 ### New Settings
 
-- `UseCompilerFolder`: Setting useCompilerFolder to true causes your pipelines to use containerless compiling. Unless you also set `doNotPublishApps` to true, setting useCompilerFolder to true won't give you any performance advantage, since AL-Go for GitHub will still need to create a container in order to publish and test the apps. In the future, publishing and testing will be split from building and there will be other options for getting an instance of Business Central for publishing and testing. 
```

Comment:
Modifying old release notes causes the Deploy to catch these changes as changes to this release.
If we want to change old release notes, we need to change this mechanism.

Reply:
What would be the effect of that?

That potentially repos running on older versions will get notified for new changes and when they update AL-Go System files, they'll see that these changes are only in the release notes?

Reply:
Added the trailing spaces back.